docs/io/fits/usage/verification.rst

```diff
@@ -246,51 +253,57 @@ Unfixable Cards:
 
 We will summarize the verification with a "life-cycle" example:
 
-.. doctest-skip::
-
     >>> h = fits.PrimaryHDU()  # create a PrimaryHDU
     >>> # Try to add an non-standard FITS keyword 'P.I.' (FITS does no allow
     >>> # '.' in the keyword), if using the update() method - doesn't work!
-    >>> h['P.I.'] = 'Hubble'
-    ValueError: Illegal keyword name 'P.I.'
+    >>> h.header['P.I.'] = 'Hubble' # doctest: +IGNORE_EXCEPTION_DETAIL
+    Traceback (most recent call last):
+     ...
+    VerifyWarning: Keyword name 'P.I.' contains characters not allowed
+        by the FITS standard; a HIERARCH card will be created.
```

Comment:
Hmm, it's a bit unfortunate that we turn warnings into errors (for unit tests) here, since for users this should give only a warning by default. I see two solutions:
- we could maybe use the `testsetup` and `testcleanup` directives to remove the 'error' filter before running this file,
- or just wrap this line with `warning.catch_warnings` to make sure warnings are not turn into exceptions here.